infrastructure/testdriver/actions/pause.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>TestDriver actions: event order</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<script>
+async_test(t => {
+  let t0 = performance.now();
+  let actions = new test_driver.Actions()
+    .addTick(2000)
+    .send()
+    .then(t.step_func_done(() => assert_greater_than(performance.now() - t0, 2000)));
```

Comment:
This test will time out instead of fail if the promise is rejected. Could you add a rejection handler or refactor it into a `promise_test`?

Reply:
done.